src/workerd/api/global-scope.c++

```diff
@@ -501,6 +501,10 @@ kj::Promise<WorkerInterface::AlarmResult> ServiceWorkerGlobalScope::runAlarm(kj:
 
         context.getMetrics().reportFailure(e);
 
+        auto description = kj::str(e.getDescription());
+        auto log = !jsg::isTunneledException(description) && !jsg::isDoNotLogException(description);
+        auto isUserError = e.getDetail(jsg::EXCEPTION_IS_USER_ERROR) != kj::none;
+
         // This will include the error in inspector/tracers and log to syslog if internal.
         context.logUncaughtExceptionAsync(UncaughtExceptionSource::ALARM_HANDLER, kj::mv(e));
```

Comment:
Nice catch... I'm surprised this problem hasn't been apparent earlier.  I suppose it would also work to move this call to the bottom of the block, or make it a `kj::cp(e)`?  One change in behavior is that we'll no longer be capturing the stack in the other logs, just the description, but not sure if that's important.

Reply:
yeah, I considered this, but all this block seems to be diagnostic only which has been ignored for a while (it doesn't have stack trace either because of mv), so it should be fine.